drivers/interrupt_controller/intc_plic.c

```diff
@@ -42,7 +42,19 @@
  * However, it is defined and supported by at least the Andes & Telink datasheet, and supported
  * in Linux's SiFive PLIC driver
  */
+#define PLIC_TRIG_LEVEL ~BIT(0)
+#define PLIC_TRIG_EDGE BIT(0)
```

Comment:
This *should* work... 
```suggestion
#define PLIC_TRIG_LEVEL ~BIT64(0)
#define PLIC_TRIG_EDGE BIT64(0)
```

Reply:
added (uint32_t) cast in the macro to fix build warning